contrib/ci/dependencies.xml

```diff
@@ -180,10 +180,12 @@
       <package>libftdi-devel</package>
     </distro>
     <distro id="debian">
+      <control />
       <package variant="x86_64" />
       <package variant="i386" />
     </distro>
     <distro id="ubuntu">
+      <control />
```

Comment:
uh, i'm not sure this can be done in Ubuntu - it's in universe according to https://launchpad.net/ubuntu/+source/libftdi
Shouldn't that mean MIR?

Reply:
Hmm, yeah - maybe it would be better to try and build libflashrom without support for FTDI seeing as it wont be used? (Cross that bridge later if needs be)

Reply:
so drop this one line and then this PR LGTM.